plugins/extractors/bigquery/bigquery.go

```diff
@@ -235,6 +233,10 @@ func (e *Extractor) buildColumn(ctx context.Context, field *bigquery.FieldSchema
 func (e *Extractor) buildPreview(ctx context.Context, t *bigquery.Table) (fields []interface{}, preview []interface{}, err error) {
 	fields = []interface{}{}  // list of column names
 	preview = []interface{}{} // rows of column values
+	previewTotalRows := e.config.TotalPreviewRows
```

Comment:
Lets use same variable naming - 
`TotalPreviewRows` and `totalPreviewRows` 
or
`PreviewTotalRows` and `previewTotalRows`.

Reply:
oh ok sure, was planning to not changing much lines. updating.

Reply:
updated

Reply:
Or why total? how about `PreviewRowCount`

Reply:
hmm to be more exact, we can even use `MaxPreviewRows`, because there can be case where the total rows in the table is less than the row count config. wdyt? @kushsharma @rohilsurana

Reply:
`MaxPreviewRows` sounds good. 👍

Reply:
updated 🙏